keys-in.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @module 101/keys-in
+ */
+
+/**
+ * Get all the keys compositing an object, including the `Object.prototype`
+ * @function module:101/keys-in
+ *
+ * @param {object} object - the object from which to extract the keys
+ * @return {array} array of keys
+ */
+module.exports = keysIn;
+
+function keysIn(object) {
+  if (!object) { return []; }
+
+  var keys = [];
+
+  if (object.prototype) { object = Object.create(object.prototype); }
```

Comment:
@tjmehta  Oh, now I see that.

Sorry for that mistake, I was stuck looking at it in the wrong way. I already corrected the problem.